src/State.ts

```diff
@@ -1,22 +1,14 @@
-/* eslint-disable eslint-comments/no-unlimited-disable */
-/* eslint-disable */
-// @ts-nocheck TODO(TS) provide types
-const State = {
+// TODO use State from RNModule
+
+export const State = {
```

Comment:
Wouldn't an enum work for this case?

Reply:
IIRC enum semantics is different from an object. I'd like to keep possibly incompatible changes outside of this PR and maybe fix them in the follow-up.